backend/src/main/java/com/google/rolecall/ApplicationLoader.java

```diff
@@ -98,50 +98,54 @@ private void createOneUser(String fName, String mName, String lName, String suff
           .setSuffix(suffix)
           .setEmail(email)
           .setDateJoined(cal)
+          .setIsAdmin(isAdmin)
           .setIsActive(true)
           .setCanLogin(true)
-          .setIsDancer(true)
+          .setIsDancer(!isAdmin)
           .build();
       userRepo.save(user);
   }
 
   private DataCreateError createTestData() {
     logger.log(Level.WARNING, "Creating test data.");    
     try {
-      createOneUser("Robert", "", "Battle", "", "rb@gmail.com","1/1/2020");
-      createOneUser("Jeroboam", "", "Bozeman", "", "bb@gmail.com","1/1/2020");
-      createOneUser("Clifton", "", "Brown", "", "cb@gmail.com","1/1/2020");
-      createOneUser("Khalia", "", "Campbell", "", "kc@gmail.com","1/1/2020");
-      createOneUser("Patrick", "", "Coker", "", "pc@gmail.com","1/1/2020");
-      createOneUser("Sarah", "", "Daley", "", "sd@gmail.com","1/1/2020");
-      createOneUser("Ghrai", "", "Devore", "", "gde@gmail.com","1/1/2020");
-      createOneUser("Solomon", "", "Dumas", "", "sdu@gmail.com","1/1/2020");
-      createOneUser("Ronni", "", "Favors", "", "rf@gmail.com","1/1/2020");
-      createOneUser("Samantha", "", "Figgins", "", "sf@gmail.com","1/1/2020");
-
-      createOneUser("James", "", "Gilmer", "", "jg@gmail.com","1/1/2020");
-      createOneUser("Vernard", "", "Gilmore", "", "vg@gmail.com","1/1/2020");
-      createOneUser("Jacqueline", "", "Green", "", "jgr@gmail.com","1/1/2020");
-      createOneUser("Jacquelin", "", "Harris", "", "jh@gmail.com","1/1/2020");
-      createOneUser("Michael", "", "Jackson", " Jr.", "mj@gmail.com","1/1/2020");
-      createOneUser("Yazzmeen", "", "Laidler", "", "yl@gmail.com","1/1/2020");
-      createOneUser("Yannick", "", "Lebrun", "", "yle@gmail.com","1/1/2020");
-      createOneUser("Constance", "Stamatiou", "Lopez", "", "csl@gmail.com","1/1/2020");
-      createOneUser("Renaldo", "", "Maurice", "", "rm@gmail.com","1/1/2020");
-      createOneUser("Corrin", "Rachelle", "Mitchell", "", "crm@gmail.com","1/1/2020");
-
-      createOneUser("Chalvar", "", "Monteiro", "", "cm@gmail.com","1/1/2020");
-      createOneUser("Belen", "", "Pereyra", "", "bp@gmail.com","1/1/2020");
-      createOneUser("Jessica", "Amber", "Pinkett", "", "jap@gmail.com","1/1/2020");
-      createOneUser("Miranda", "", "Quinn", "", "mq@gmail.com","1/1/2020");
-      createOneUser("Jamar", "", "Roberts", "", "jr@gmail.com","1/1/2020");
-      createOneUser("Matthew", "", "Rushing", "", "mr@gmail.com","1/1/2020");
-      createOneUser("Kanji", "", "Segawa", "", "ks@gmail.com","1/1/2020");
-      createOneUser("Courtney", "Celeste", "Spears", "", "ccs@gmail.com","1/1/2020");
-      createOneUser("Jermaine", "", "Terry", "", "jt@gmail.com","1/1/2020");
-      createOneUser("Christopher", "", "Wilson", "", "cw@gmail.com","1/1/2020");
-
-      createOneUser("Brandon", "", "Woolridge", "", "bw@gmail.com","1/1/2020");
+      createOneUser("Robert", "", "Battle", "", "rb@gmail.com","1/1/2020", true);
```

Comment:
I have concerns about submit these email to the public repo. Please reconsider.

Reply:
That's a great point, in fact these are already submitted. This might be a privacy violation by posting all users emails publicly.